packages/i18next/README.md

```diff
@@ -49,16 +49,8 @@ npm install @sapphire/plugin-i18next @sapphire/framework @sapphire/pieces@1 disc
 import '@sapphire/plugin-i18next/register';
 ```
 
-And for discord.js:
-
-```typescript
-import '@sapphire/plugin-i18next/register-discordjs';
-```
-
 It is to be noted that unless you are using discord.js, which has the convenience register to extend the client, guild, channel and message methods for you, you will have to implement your own extensions.
```

Comment:
This should likely be removed now that we depend on discord.js as it would be impractical to use other things.